src/borg/platform/__init__.py

```diff
@@ -1,11 +1,11 @@
-import sys
-
 """
 Platform-specific APIs.
 
 Public APIs are documented in platform.base.
 """
 
+from ..platformflags import is_win32, is_linux, is_freebsd, is_darwin
```

Comment:
Because this produces circular references.

Think about the usage if you put this in borg.platform.flags / borg.platform.__init__ or you can't Import anything from outside of borg.platform from there… it will result in a exception.